godaddy-wordpress/coblocks
# Fix CoBlocks for Gutenberg 7.2.0: replace legacy editor- classes with block-editor-

Gutenberg 7.2.0 has removed the legacy `editor-` classes in https://github.com/WordPress/gutenberg/pull/19050 breaking the CoBlocks styles. This PR updates all those classes to use the `block-editor-` prefix.

**Screenshots**

| Before | After |
|---|---|
| <img width="347" alt="Screenshot 2020-01-09 at 14 35 24" src="https://user-images.githubusercontent.com/1233880/72072246-d0351880-32ed-11ea-80ea-f260ab5f057f.png"> | <img width="664" alt="Screenshot 2020-01-09 at 14 34 06" src="https://user-images.githubusercontent.com/1233880/72072257-d75c2680-32ed-11ea-9c79-25588daa446c.png"> |
| <img width="362" alt="Screenshot 2020-01-09 at 14 35 29" src="https://user-images.githubusercontent.com/1233880/72072269-dfb46180-32ed-11ea-8fd3-867a382c30db.png"> | <img width="357" alt="Screenshot 2020-01-09 at 14 34 10" src="https://user-images.githubusercontent.com/1233880/72072283-e4791580-32ed-11ea-9334-0a2f1bb8f2f6.png"> |
| <img width="565" alt="Screenshot 2020-01-09 at 14 35 33" src="https://user-images.githubusercontent.com/1233880/72072304-efcc4100-32ed-11ea-8869-59385f4bcc46.png"> | <img width="643" alt="Screenshot 2020-01-09 at 14 34 14" src="https://user-images.githubusercontent.com/1233880/72072312-f5298b80-32ed-11ea-8a8d-157e529af0bc.png"> | 

**How to test**
- Activate Gutenberg 7.2.0
- Generate a new build of CoBlocks using this branch and activate it.
- Verify the blocks look correct.
